index.md

```diff
@@ -11,26 +11,67 @@ layout: default
 
 # Apache Beam (incubating)
 
-Apache Beam is an open source, unified model and set of language-specific SDKs for defining data processing workflows that may then be executed on top of a set of supported runners, currently including [Apache Flink](http://flink.apache.org), [Apache Spark](http://spark.apache.org), and [Google Cloud Dataflow](https://cloud.google.com/dataflow).
+Apache Beam is an open source, unified programming model that you can use to define data processing workflows. You define a workflow, called a **pipeline**, by building a program with a Beam SDK. The pipeline is then executed by one of Beam's supported **distributed processing back-ends**, which include [Apache Flink](http://flink.apache.org), [Apache Spark](http://spark.apache.org), and [Google Cloud Dataflow](https://cloud.google.com/dataflow).
```

Comment:
Do we need a distrinction between workflow and pipeline? How about the following?

Apache Beam is an open source, unified programming model that you can use to define data processing pipelines. You start by build a pipeline with one of the Beam SDKs, and then execute that pipeline on one of Beam's supported **distributed processing back-ends**, which include...